helm-chart/templates/nextcloud-aio-nextcloud-data-persistentvolumeclaim.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  creationTimestamp: null
+  labels:
+    io.kompose.service: nextcloud-aio-nextcloud-data
+  name: nextcloud-aio-nextcloud-data
+spec:
+  accessModes:
```

```suggestion
spec:
  storageClassName: {{ .Values.STORAGE_CLASS }}
  accessModes:
```

Let admins configure their storage classes. This also applies to other manifests.

This should probably be guarded by an if condition. The default storage class will only be applied if the `storageClassName` is missing.

---

done